File: model/src/main/java/no/nav/pensjon/vtp/testmodell/personopplysning/PersonModellRepository.java
package no.nav.pensjon.vtp.testmodell.personopplysning;

import java.util.Optional;

import org.springframework.data.repository.Repository;

public interface PersonModellRepository extends Repository<PersonModell, String> {
    Optional<PersonModell> findById(String ident);

    Optional<PersonModell> findByAktørIdent(String ident);

    PersonModell save(PersonModell bruker);
}

File: core/src/main/kotlin/no/nav/pensjon/vtp/testmodell/personopplysning/PersonModellRepository.kt
package no.nav.pensjon.vtp.testmodell.personopplysning

import org.springframework.data.repository.Repository

interface PersonModellRepository : Repository<PersonModell, String> {
    fun findById(ident: String): PersonModell?
    fun findByAktørIdent(ident: String): PersonModell?
    fun findAll(): List<PersonModell>
    fun save(bruker: PersonModell): PersonModell?
}
